apps/src/aichat/views/ChatMessageView.tsx

```diff
@@ -44,9 +45,16 @@ const ChatMessageView: React.FunctionComponent<ChatMessageViewProps> = ({
   // but is currently marked optional because the ChatEvent type
   // is used for both chat history and live chat.
   // TODO: Clean up types to separate server and client IDs.
-  const commonProps = {...chatMessage, id: chatMessage.id!};
-  const footer = messageVisible ? (
-    <CleanFeedbackFooter {...commonProps} />
+  const commonProps = {
```

Comment:
This slims down the `commonProps` object to only include actually shared props, rather than every property on the `chatMessage` object.